todo/urls.py

```diff
@@ -32,4 +34,5 @@
     path("complete/<int:todo_id>/", complete_todo, name="complete_todo"),
     path("delete/<int:todo_id>/", delete_todo, name="delete_todo"),
     path("clear/", clear_todos, name="clear_todos"),
+    path("todos.json", list_todos_json, name="list_todos_json"),
```

Comment:
`todos.json` in the URL is a bit unusual from a Django point of view. More typical would be something like `api/todos/`.